src/main/groovy/org/pih/warehouse/outbound/ImportPackingListItem.groovy

```diff
@@ -56,6 +56,15 @@ class ImportPackingListItem implements Validateable {
             // returns a dummy location object to add more context in a response of what location was not found
             return new Location(name: source['binLocation'])
         }
+
+        Product product = Product.findByProductCode(source['product'])
```

Comment:
can't we use `obj.product` at this point? If it is `null` at this point, we would just have to change the order of the payload that is sent to the API,, so that the product appears before the binLocation.

Reply:
yes it is null at this point.
I would avoid resorting to this type of a solution, feels a little hacky that a client has to send a payload in a specific order